<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buzzer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/img/ressources/Question.png</normaloff>:/img/ressources/Question.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboPort">
         <property name="maximumSize">
          <size>
           <width>121</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>116</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Vitesse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboVitesse"/>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonConnexion">
         <property name="text">
          <string>Connexion</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Emission</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="boxEmission"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>RÃ©ception</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="boxReception"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/img/ressources/exit.png</normaloff>:/img/ressources/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboPort</tabstop>
  <tabstop>comboVitesse</tabstop>
  <tabstop>ButtonConnexion</tabstop>
  <tabstop>boxEmission</tabstop>
  <tabstop>boxReception</tabstop>
 </tabstops>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
